pkgs/applications/networking/instant-messengers/nheko/default.nix

```diff
@@ -23,8 +23,22 @@
 , olm
 , pkg-config
 , nlohmann_json
+, voipSupport ? true
+  , gst_all_1
+  , libnice
```

Comment:
```suggestion
, gst_all_1
, libnice
```

Reply:
This is on purpose: it's to distringuish the optional packages, see this expression also [mpv-unwrapped](https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/video/mpv/default.nix#L7).

Reply:
Packages after an option usually belong to it. You can use new lines to make that clear. Also mpv is not a good reference here because no one changed that line in 5 years and you introduced that formatting https://github.com/NixOS/nixpkgs/commit/6fabc515d01a5644a027a7d12f845979fce023a3 .

I would like to see that removed to make treewide changes easier where this formatting is not assumed.

Reply:
>  Also mpv is not a good reference here because no one changed that line in 5 years and you introduced that formatting 6fabc51 .

Indeed, I mentioned mpv because I contributed, but it's not a pattern I came up with.

> I would like to see that removed to make treewide changes easier where this formatting is not assumed.

Changing the formatting just so that a regex can perform changes does not seem like a sensible idea.
It should be the other way around, to me.

Reply:
I think having consisten formating that is important, and if it helps automation even more so.

Reply:
Done, more opaque and bots and `sed` love it. Now if we could stop focusing on stylistic changes...